python/dolfinx/wrappers/io.cpp

```diff
@@ -26,11 +26,58 @@
 #include <string>
 #include <vector>
 
+#include <petscsystypes.h>
```

Comment:
I find this objectionable - I/O has nothing to do with PETSc. Can't this be fixed with a little more templating?

Reply:
Fixed.